README.md

```diff
@@ -71,6 +71,16 @@ iex -S mix phx.server
 iex -S mix
 ```
 
+### Authorization
+
+By default, Castle will restrict what podcasts you can see based on the
+account-ids granted to you by [ID](https://github.com/PRX/id.prx.org). If you
+want to impersonate other accounts, just set `DEV_AUTH=123,456,789` in your
+ENV to grant you access to that comma-separated list of account ids. You can
+also set `DEV_AUTH=*` to allow access to all accounts.
+
+Note that the `DEV_AUTH` ENV does not work at all in production environments.
+
```

Comment:
nice dev feature! Seems like I'm only getting access to podcasts that the DEV_AUTH account actually owns rather than all podcasts they have access to. When I have my `DEV_AUTH` set to admin, I only see the Dog Talk podcast. But I get everything when set to `*`.

Reply:
Hmmm... so you're a non-owner of an account, and not seeing that podcast in your auth?

Yeah, I was expecting ID would give Metrics a token containing _all_ your account-ids.  If that's not true, could definitely cause problems for users.  Will look into it.

Reply:
Oh wait, this isn't even involving ID/Metrics.  Hmm...

Reply:
Yeah, i guess that's what i'd expect.  In real life, your auth-token from ID will contain any account-ids you have access to (not just accounts you're an admin in).  And castle does a dumb comparison of `podcast.account_id` against that list.  Probably worth double-checking that in staging before sending this to prod though.  Would hate to lock a bunch of users out.